src/backend/distributed/commands/extension.c

```diff
@@ -655,12 +654,12 @@ GetAllViews(void)
 
 
 /*
- * ShouldMarkRelationDistributedOnUpgrade is a helper function that
+ * ShouldMarkRelationDistributed is a helper function that
  * decides whether the input relation should be marked as distributed
```

Comment:
Stale comment. Besides, we may want to move that function to distobject.c

Reply:
makes sense, done